packages/core/types/report.d.ts

```diff
@@ -33,7 +33,14 @@ declare class Report {
     url: string;
   };
 
-  constructor(errorClass: string, errorMessage: string, stacktrace?: any[], handledState?: IHandledState);
+  constructor(
+    errorClass: string,
+    errorMessage: string,
+    stacktrace?: any[],
+    handledState?: IHandledState,
+    originalError?: any,
```

Comment:
Is there some docs or usage/tests to go along with this change?

Reply:
I figured I would write the docs once the implementation was ratified. I don't have a good reason for not adding for that aspect of this PR though so I will add some.